quiche/src/h3/mod.rs

```diff
@@ -552,6 +554,13 @@ impl Config {
     pub fn set_qpack_blocked_streams(&mut self, v: u64) {
         self.qpack_blocked_streams = Some(v);
     }
+
+    /// Sets or omits the `SETTINGS_ENABLE_CONNECT_PROTOCOL` setting.
+    ///
+    /// The default value is `0`.
+    pub fn set_connect_protocol_enabled(&mut self, v: u64) {
```

Comment:
this would probably be better modelled as a bool such as `enable_extended_connect(enabled: bool)`

Reply:
modelling the setting internally as an uint is fine, but there's little reason for us to let users of quiche set random values when the RFC doesn't permit that

Reply:
thanks, addressed in ced7972025b1c0993642df69fa09520713d320bc